.github/ISSUE_TEMPLATE/3-new-model.yml

```diff
@@ -1,6 +1,6 @@
 name: "\U0001F31F New model/dataset/scheduler addition"
 description: Submit a proposal/request to implement a new model / dataset / scheduler
-labels: [ "new-models" ]
+labels: [ "feature-request" ]
 title: "[New Models] "
```

Comment:
Maybe the title should be changed correspondingly?

Reply:
I think we can use this prefix to distinguish this kind of issue from the one by using a feature-request template. 
both new models/datasets or other new features will be treated as "feature-request" in our daily development.